extensions/amp-bind/0.1/test/integration/test-bind-impl.js

```diff
@@ -1138,6 +1138,109 @@ describe
           yield onBindReadyAndSetState(env, bind, {foo: 'bar'});
           expect(foo.textContent).to.not.equal('bar');
         });
+
+        describe('AmpEvents.VALUE_CHANGE dispatches', () => {
```

Comment:
nit: remove "AmpEvents.VALUE_CHANGE"?

It sounds more like a sentence "AmpEvents.VALUE_CHANGE" and 'should dispatch on <input [value]> changes'

Reply:
Made the test name more elaborate so there is no confusion "should dispatch VALUE_CHANGE on ..."